Source/Core/Runtime/Attributes/DefaultImplementationAttribute.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace VRBuilder.Core.Attributes
+{
+	[AttributeUsage(AttributeTargets.Class)]
+	public class DefaultImplementationAttribute : Attribute
```

Comment:
I would rename this to DefaultSceneObjectPropertyAttribute or something similar, the point being that it only works on properties and DefaultImplementation sounds too general.